### ISSUE TITLE
Examples Page

### ISSUE THREAD

**Issue by loganfranken** [UCSantaBarbara/webguide#34]
In lieu of the Three Tiers page (#1), the group discussed creating an "Examples" page that would put an emphasis on high-profile/high-traffic websites that should adhere to Web Standards Guide best practices.

**Comment by loganfranken** [comment]
I don't mind throwing this together, but the biggest sticking point is that "Examples" is already taken.

Does someone have a better name for this page? Or should we maybe rename the current "examples" repo to "templates"?

**Comment by garster** [comment]
I reviewed and concur with @loganfranken

**Comment by UCEAP-JCOLON** [comment]
Yup, I agree:
Examples to new templates page and have a new examples page which includes "tiers".

**Comment by loganfranken** [comment]
Alright, I propose the following content for this "Examples" page:

## 

**Title:** Examples
**URL:** /examples

The following websites are examples of high-traffic, high-profile websites that adhere to the best practices outlined in the Web Standards Guide:

### Campus-wide Services
- [Admissions](http://admissions.sa.ucsb.edu/)
- [Office of the Registrar](http://registrar.sa.ucsb.edu/)
- [Library](http://www.library.ucsb.edu/)
- [Division of Student Affairs](http://www.sa.ucsb.edu/)
- [Graduate Division](http://www.graddiv.ucsb.edu/)

### Major Academic Units
- [College of Engineering](http://engineering.ucsb.edu/)
- [College of Letters and Science](http://www.ltsc.ucsb.edu/)
- [College of Creative Studies](https://ccs.ucsb.edu/)

## 

Basically: I just kept the Tier 1 websites as I felt the choices for the remaining tiers were an arbitrary sampling. Of course, this isn't a definitive list and we can build out this section over time (maybe even integrate the WSG Site of the Month)?

I removed [GOLD](https://my.sa.ucsb.edu/gold/login.aspx) since it's behind a login so it doesn't serve as a very good reference.

**Comment by tenken** [comment]
This all looks good to me (and I made 1 of those sites, so yay!?)
